src/viam/resource/easy_resource.py

```diff
@@ -25,6 +28,50 @@ def _parse_model(orig: Union[str, Model]) -> Model:
     return Model(ModelFamily(*family), name)
 
 
+def _create_stub_fn(name: str, is_async: bool) -> Callable:
+    """
+    This creates a sync or async stub function which returns a MethodNotImplementedError.
+    The stub_model decorator uses these to stub out an abstract base class.
+    """
+    if is_async:
+        # note: this is a pyright bug https://github.com/microsoft/pyright/issues/2136
+        async def stub_fn(*args, **kwargs):  # pyright: ignore [reportRedeclaration]
+            logger.info(f'{name} not implemented')
```

Comment:
Do we need the log statements? Seems redundant with the error type

Reply:
there are some edge cases in the Motor control card that don't get toast popups in app

(one method that is required to even display the motor RC card, another that is being polled in the background all the time; I think one is get properties and the other is is_moving)